test/e2e/kubernetes/node/node.go

```diff
@@ -47,17 +47,11 @@ func GetWindowsVersion(ctx context.Context, clientset *kubernetes.Clientset) (wi
 		LabelSelector: "kubernetes.io/os=windows",
 	}
 	var result *corev1.NodeList
-	Eventually(func() error {
+	Eventually(func(g Gomega) {
 		var err error
 		result, err = clientset.CoreV1().Nodes().List(ctx, options)
-		if err != nil {
-			return err
-		}
-
-		if len(result.Items) == 0 {
-			return fmt.Errorf("no Windows Nodes found")
-		}
-		return nil
+		g.Expect(err).NotTo(HaveOccurred())
+		g.Expect(len(result.Items)).NotTo(Equal(0))
```

Comment:
super nit

```suggestion
		g.Expect(result.Items).NotTo(BeEmpty())
```

is slightly more readable